javascript/javascript2/week3/warmup.js

```diff
@@ -0,0 +1,96 @@
+//1.Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+
+setTimeout(function () {
+	console.log("Called after 2.5 seconds after the script is loaded");
+}, 2500);
+
+//2.Create a function that takes 2 parameters: delay and stringToLog.
```

Comment:
I see huge number of line for 'commenting only', this is totally fine. but better coding practice please read about 'JSDoc'. check some youtbe tutorials.

Reply:
Thank you for the review. i will read 'JSDoc' and update my code.